SPDLOG_QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,90 @@
+This document is a declaration of software quality for the `spdlog` external dependency imported by the `spdlog_vendor` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `spdlog` Quality Declaration
+
+TODO: Declare the quality level for `spdlog`
+
+## Version Policy
```

Comment:
We've updated the format a bit to include section numbers. See [rcpputils for example](https://github.com/ros2/rcpputils/blob/master/QUALITY_DECLARATION.md)